docs/standards/logging_standards.md

```diff
@@ -0,0 +1,78 @@
+# Logging standards
+
+Logging is a critical part of any application. It provides a way to monitor the health of the application, debug issues, and provide an audit trail of what has happened.
+
+## Only log useful information
+
+This will depend on the application, but some common things to log include:
+- **Errors**: Capture unexpected issues or failures.
+- **Warnings**: Highlight potential problems that may not immediately impact functionality.
+- **Informational messages**: Provide insights into the application's normal operations.
+- **Debug messages**: Help developers understand the application's behavior during development.
+- **Audit logs**: Record user actions or system events for compliance and security purposes.
+
+Logging excessive or redundant information can make logs harder to analyze and increase storage costs.
+
+Consider the logging any third-party libraries do and configure them appropriately.
+
+## Log different levels across environments
+
+Different log levels are useful in different environments. Use the following as a baseline approach and ensure that log levels are easily configurable:
+
+- **Development**: Log debug messages to assist with troubleshooting.
+- **Testing/Staging**: Focus on warnings and errors to validate application behavior.
+- **Production**: Restrict logs to errors and critical warnings to minimize noise and protect performance.
+
+
```

Comment:
Additional blank line.

I thought we had linting checks for stuff like this at one point, but I can't see any evidence of it so maybe I dreamt it?